apis/keda/v1alpha1/scaledobject_webhook_test.go

```diff
@@ -486,6 +486,68 @@ var _ = It("should validate so creation when min replicas is > 0 with only cpu s
 
 })
 
+var _ = It("should not validate ScaledObject creation when deployment only provides cpu resource limits", func() {
+
+	namespaceName := "only-cpu-resource-limits-set"
+	namespace := createNamespace(namespaceName)
+
+	err := k8sClient.Create(context.Background(), namespace)
+	Expect(err).ToNot(HaveOccurred())
+
+	workload := createDeployment(namespaceName, false, false)
+	workload.Spec.Template.Spec.Containers[0].Resources.Limits = v1.ResourceList{
+		v1.ResourceCPU: *resource.NewMilliQuantity(100, resource.DecimalSI),
+	}
+
+	err = k8sClient.Create(context.Background(), workload)
+	Expect(err).ToNot(HaveOccurred())
+
+	so := createScaledObject(soName, namespaceName, workloadName, "apps/v1", "Deployment", false, map[string]string{}, "")
+	so.Spec.Triggers = []ScaleTriggers{
+		{
+			Type: "cpu",
+			Metadata: map[string]string{
+				"cpu": "10",
+			},
+		},
+	}
```

Comment:
This isn't part of the CPU Scaler spec: https://keda.sh/docs/2.11/scalers/cpu/
I'm not totally sure about what you wanted to set here

Reply:
Oh damn it... I missed it. 🙄